cinn/frontend/op_mappers/paddle/arg_min_max.cc

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2022 CINN Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "cinn/frontend/op_mapper_registry.h"
+#include "cinn/frontend/op_mappers/common_utils.h"
+#include "cinn/frontend/var_type_utils.h"
+
+namespace cinn {
+namespace frontend {
+namespace paddle_mappers {
+
+void ArgOpMapperHelper(const paddle::cpp::OpDesc& op_desc, const OpMapperContext& ctx, const std::string arg_type) {
+  CHECK_EQ(op_desc.Input("X").size(), 1UL);
+  auto x_name = op_desc.Input("X").front();
+
+  CHECK_EQ(op_desc.Output("Out").size(), 1UL);
+  auto out_name = op_desc.Output("Out").front();
+
+  auto x        = ctx.GetVar(x_name);
+  auto axis     = op_desc.GetAttr<int32_t>("axis");
+  auto keepdims = op_desc.GetAttr<bool>("keepdims");
+  auto flatten  = op_desc.GetAttr<bool>("flatten");
```

Comment:
这三个在 Paddle 里面是有默认值的：

```suggestion
  auto axis     = op_desc.GetAttr<int32_t>("axis");
  auto keepdims = op_desc.GetAttr<bool>("keepdims");
  auto flatten  = op_desc.GetAttr<bool>("flatten");
```

Reply:
我觉得可以增加check，而不是添加默认值，毕竟这里只是paddle到cinn的mapper，不是一个单独的op或者函数，如果paddle本身传过来的部分attr属性缺失，这里应该报错才对吧

Reply:
同样建议使用`GetAttrOrDefault`替代`op_desc.GetAttr`，如果觉得应该报错的话在之前加个CHECK：
```
CHECK(op_desc.HasAttr("axis")) << arg_type << " op should has attribute \"axis\"! Please check.";
```

`GetAttrOrDefault`不仅仅是设置默认值的问题，比如对于CINN python测试中传入的空list属性值，如果直接`GetAttr`是会报错的，但某些情况下空list的测试又是非常重要的，`GetAttrOrDefault`可以协助处理，避免报错。

Reply:
ok